content/en/blog/2022/community-meeting-february-2022.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: blog
+title: "February 2022 Community Meeting Highlights"
+date: 2022-04-28
+slug: community-meeting-februrary-2022
+---
+
+**Author:** Nigel Brown (VMware)
+
+We just had our first contributor community meeting this year, and it was awesome to be back with you 
+in that format. These meetings will be happening on Zoom once per month, on the third Thursday of the 
+month - that should be available in your calendar if you’re subscribed to the k-dev mailing list. 
+Community meetings are an opportunity for you to meet synchronously with other members of the 
+Kubernetes community to talk about issues of general appeal.
+
+This meeting kicked off with an update on the 1.24 release with Xander Grzywinski, who is one of the 
+shadows for the release team leads. This release is scheduled for April 19, 2022 with a code freeze 
```

Comment:
I think we should add about the delay here too

Reply:
At the February community meeting, the v1.24 release was not delayed at that moment.
Is it appropriate for the Feb Community Meeting blog?

I will have a snippet about the v1.24 delay in the April Community Meeting blog which I will author after this is published

Reply:
Or we can have an __Update__ part or  a __Note__ part in this blog

Reply:
I somewhere think we should add a update or note to notify the truth maybe, will wait for thoughts of other folks on this

Reply:
Yeah, I think a footnote is appropriate, just to not confuse people.

Reply:
Added an _Update_ to the blog and squashed commits